components/style/traversal.rs

```diff
@@ -220,20 +244,30 @@ pub fn recalc_style_at<'a, N, C>(context: &'a C,
             StyleSharingResult::CannotShare => {
                 let mut applicable_declarations = ApplicableDeclarations::new();
 
+                let relations;
                 let shareable_element = match node.as_element() {
                     Some(element) => {
+                        if opts::get().style_sharing_stats {
```

Comment:
`opts` is not the same as `prefs`, and `opts::get()` is just getting a lazy-static, not a `RWLock` as with the preferences.